packages/preview/numberingx-0.0.1/typst.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name        = "numberingx"
+version     = "0.0.1"
+description = "Extended numbering patterns using the CSS Counter Styles spec"
+license     = "MIT-0"
+authors     = ["Edhebi"]
+repository  = "https://github.com/edhebi/numberingx"
+entrypoint  = "numberingx.typ"
+exclude     = [".git*"]
```

Comment:
.git is ignored by default

Reply:
that isn't the .git folder, it's for the gitattribute and .gitignore I have in the repo, but in the end I just didn't copy them over to the package registry to begin with.